### PR TITLE
chore: bump celestia app v0.8.0

### PR DESCRIPTION
**Pull request by evan-forbes**
- bumps celestia-app to v0.8.0, which includes a new default token ticker, along with 512 byte shares (8MB block)
- increases the integration test block times to accommodate the bigger blocks (we had to do this in celestia-app too)

### CODE REVIEW

**Review by Wondertan — APPROVED** [review]

File: nodebuilder/tests/swamp/config.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ type Components struct {
 // In addition, the empty block interval is set to 50ms
```
Comment (Wondertan) [inline nodebuilder/tests/swamp/config.go:20]: nit: can you remove this comment pls

**Review by renaynay — APPROVED** [review]

**Comment by Wondertan** [comment]
@evan-forbes ping to update this to further version

**Comment by Wondertan** [comment]
Superseded by #1300

File: nodebuilder/tests/swamp/config.go (hunk 1 above)
Comment (evan-forbes) [reply nodebuilder/tests/swamp/config.go]: :+1: https://github.com/celestiaorg/celestia-node/pull/1240/commits/47ca95443d6422af49ac4678844d5cb61f04693c

**Comment by evan-forbes** [comment]
`TestReadEDS` is failing, and I think we might have to regenerate the test file used for reading CAR to accommodate for the larger square size.

**Comment by Wondertan** [comment]
@distractedm1nd, ^

**Comment by evan-forbes** [comment]
closing in favor of v0.9.0 #1300

**Pull request closed without merging**